index.html

```diff
@@ -61,5 +61,203 @@ <h1>
         </li>
       </ul>
     </section>
+    
+    <section class="work-section">
+      <article class="portfolio-item">
+        <img src="./assets/images/tonic.png" alt="tonic" class="mt-12" />
+        <div class="title-block mt-12">
+          <h2 class="project-title">Tonic</h2>
+          <div class="project-role">
+            <p class="client">CANOPY</p>
+            <img src="./assets/Icons/Counter.svg" alt="seperator" />
+            <p class="role">Back End Dev</p>
+            <img src="./assets/Icons/Counter.svg" alt="seperator" />
+            <p class="year">2015</p>
+          </div>
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tech-tags">
+          <li class="tech-tag-name"><span class="tech-tag-text">html</span></li>
```

Comment:
- Good job using `flex` to style your project cards 👏 but there are still some issues that need your attention.
- [ ] You'll notice that the background around your project cards is different from the white background of the cards themselves. Kindly address this issue so as to follow the template design.
- [ ] Also your technologies have the wrong color. It would be good to use the right color from the template 👍 
- Your design:
![Capture3](https://user-images.githubusercontent.com/85113567/154784241-6aa43bee-b06f-4428-a850-59599cb9de6b.PNG)
- Figma design:
![Capture5](https://user-images.githubusercontent.com/85113567/154784250-4b0b9a29-7dd4-42a9-91ff-f190ff07d33b.PNG)